app/services/TouchpointBackends.scala

```diff
@@ -87,13 +77,13 @@ class TouchpointBackends(actorSystem: ActorSystem, wsClient: WSClient, execution
       lazy val environmentName: String = config.environmentName
       lazy val salesforceService = new SalesforceServiceImp(sfSimpleContactRepo)
       val catalogRestClient: SimpleClient[Future] = new rest.SimpleClient[Future](config.zuoraRest, configurableFutureRunner(60.seconds))
-      lazy val catalogService = new subsv2.services.CatalogService[Future](newProductIds, catalogRestClient, Await.result(_, 10.seconds), backendType.name)
+      lazy val catalogService = new subsv2.services.CatalogService[Future](newProductIds, FetchCatalog.fromZuoraApi(catalogRestClient), Await.result(_, 10.seconds), backendType.name)
```

Comment:
This should preserve the existing behaviour (it's needed due to an earlier change in membership-common - https://github.com/guardian/membership-common/pull/576 - nothing to do with metrics)